web/src/layout/package/CapabilityLevel.tsx

```diff
@@ -5,7 +5,7 @@ import { FaCheck } from 'react-icons/fa';
 import { OPERATOR_CAPABILITIES } from '../../utils/data';
 import SmallTitle from '../common/SmallTitle';
 import styles from './CapabilityLevel.module.css';
-import CapatabilityLevelInfoModal from './CapatabilityLevelInfoModal';
+import CompatibilityLevelInfoModal from './CompatibilityLevelInfoModal';
```

Comment:
This family was renamed.
Upon review, I think I let Google Sheets guess the name incorrectly.

Capatability | =
-|-
Capability | ^
Compatibility | v

Reply:
Capability is ok